.gitignore

```diff
@@ -49,9 +49,10 @@ Apromore-Assembly/Parquet-Repository
 Apromore-Assembly/virgo-tomcat-server-3.6.4.RELEASE
 
 Event-Logs-Repository/
-# Ignore Gradle project-specific cache directory
-.gradle
 
+# Ignore Gradle project-specific cache directory
+.gradle/
 # Ignore Gradle build output directory
-build
+**/build/*
```

Comment:
Is this intended?

Reply:
Just for consistency with the earlier pattern :)